### ISSUE TITLE
dinf-tabulate broken pipe when piping output to head

### ISSUE THREAD

**Issue by grahamgower** [RacimoLab/dinf#325]
```
$ dinf-tabulate data docs/guide/out/bottleneck-smc/19/smc.npz |head
_Pr     N0      N1
6.693139076232910156e-01        9.904836838507975699e+03        2.540576416713277581e+02
5.270755887031555176e-01        1.048883177066255121e+04        1.313702521022924543e+02
8.349320888519287109e-01        9.756428207835488138e+03        2.176912296627198202e+02
8.926991224288940430e-01        1.027066328428442102e+04        2.714284271946929152e+02
2.640830874443054199e-01        9.609516241364051893e+03        2.538017319184321536e+02
7.795472145080566406e-01        1.064387728001731375e+04        8.232987931949718075e+01
7.977599501609802246e-01        1.029228826017841857e+04        1.441362585085516343e+02
8.896112442016601562e-01        1.083200331833681594e+04        2.460527824468115341e+02
7.925135493278503418e-01        9.426981129653318931e+03        2.185261126100251658e+02
Traceback (most recent call last):
  File "/home/grg/src/dinf/venv/bin/dinf-tabulate", line 8, in <module>
    sys.exit(main())
  File "/home/grg/src/dinf/src/dinf/tabulate.py", line 236, in main
    args.func(args)
  File "/home/grg/src/dinf/src/dinf/tabulate.py", line 176, in __call__
    np.savetxt(f, data, delimiter=args.separator, fmt=args.format)
  File "<__array_function__ internals>", line 200, in savetxt
  File "/home/grg/src/dinf/venv/lib/python3.10/site-packages/numpy/lib/npyio.py", line 1613, in savetxt
    fh.write(v)
  File "/home/grg/src/dinf/venv/lib/python3.10/site-packages/numpy/lib/npyio.py", line 1525, in write_normal
    self.fh.write(asunicode(v))
BrokenPipeError: [Errno 32] Broken pipe
```

**Comment by grahamgower** [comment]
Recommended way to handle this:
https://docs.python.org/3/library/signal.html#note-on-sigpipe

**Comment by grahamgower** [comment]
Closed by #326.
